Backend/BikesRentalServer.Tests/UsersService/BlockUser.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BikesRentalServer.DataAccess;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services;
+using BikesRentalServer.Tests.Mock;
+using FluentAssertions;
+using Xunit;
+
+namespace BikesRentalServer.Tests.UsersService
+{
+    public class BlockUser
+    {
+        private readonly DatabaseContext _dbContext;
+        private readonly Services.UsersService _usersService;
+
+        public BlockUser()
+        {
+            _dbContext = MockedDbFactory.GetContext();
+            _usersService = new Services.UsersService(_dbContext);
+        }
+
+        [Fact]
+        public void BlockUserThatDoesntExist()
```

Comment:
Dotychczas stosowaliśmy konwencję, że w nazwie piszemy też co testowana metoda ma zwrócić w danych warunkach (`BlockUserThatDoesntExistShouldReturnEntityNotFound`), trochę pierdoła ale summon @piotrduperas dla zaopiniowania

To się tyczy obu plików z testami